.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        version: ['5.15.2']
+        version: ['6.1.0']
     steps:
```

Comment:
i believe 6.2.0 is available as preview now, not sure if it's already supported by your actions

Reply:
Yeah, lemme try it out

Reply:
> i believe 6.2.0 is available as preview now, not sure if it's already supported by your actions

Yes, seems the build has passed.